db/migrate/20170309152057_add_data_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddDataToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :gender, :string, default: 'Not defined'
```

Comment:
It was my first thing how should look new user which didn't set gender.
Later I though.. What if user wants to change gender from (sth) female to Not defined. I forgot about this problem..
It will be better if I change this migration :) thanks